JS/Mery/Health-App/src/containers/MainContainer.jsx

```diff
@@ -0,0 +1,106 @@
+import React from 'react'
+import { findDOMNode } from 'react-dom'
+import { Switch, Route, withRouter } from 'react-router-dom'
+
+import {
+  Card,
+  Divider,
+  withResize,
+  overrideThemeVariables
+} 
+from 'ui-neumorphism'
+import 'ui-neumorphism/dist/index.css'
+
+import Topbar from '../containers/Topbar'
+import routes from '../routes';
+
+class MainContainer extends React.Component {
+  mainView = null
+
+  constructor(props) {
+    super(props)
+    this.state = {
+      dark: false,
+      open: false
+    }
+    this.toggleTheme = this.toggleTheme.bind(this);
+  }
+
+  get isHome() {
+    const { pathname: p } = this.props.location
+    return (
+      p === '/' || p === '/home' || p === '/examples' || p === '/typography'
```

Comment:
I dont see where 'typography' or 'examples' path is defined. I assume you dont need them. Please recheck and understand for what you use those

Reply:
Yes I do not need them. Fixed!!